### PR TITLE
Update claude extension

### PR DESCRIPTION
**Pull request by alejandrotercero**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/claude/README.md [hunk 1]
```diff
@@ -44,6 +44,10 @@ Got the answer that you wanted? Great.
 
 # Models available
 
+## Calude 3.5
```
Comment (florisdobber) [inline extensions/claude/README.md:47]: Can you just fix this spelling error?

**Comment by florisdobber** [comment]
@alejandrot96 So quick, thanks for implementing! If you fix the spelling error, it'll be perfect!

File: extensions/claude/README.md (hunk 1 above)
Comment (alejandrot96) [reply extensions/claude/README.md]: Done

**Review by florisdobber — APPROVED** [review]
LGTM!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**